The following modules were not unloaded:
  (Use "module --force purge" to unload all):

  1) slurm/23.02.6
/lustre/home/ah1365/.local/lib/python3.9/site-packages/e3nn/o3/_wigner.py:10: UserWarning: Environment variable TORCH_FORCE_NO_WEIGHTS_ONLY_LOAD detected, since the`weights_only` argument was not explicitly passed to `torch.load`, forcing weights_only=False.
  _Jd, _W3j_flat, _W3j_indices = torch.load(os.path.join(os.path.dirname(__file__), 'constants.pt'))
Traceback (most recent call last):
  File "/lustre/projects/Research_Project-T127870/ah1365/task/investigate_artimus/SiGe/20250404210804/DRUN_MLP_MACE/20250427231949/D44/DSHIFT/D05/run-mace.py", line 7, in <module>
    from mace.calculators import mace_mp
  File "/lustre/home/ah1365/.local/lib/python3.9/site-packages/mace/calculators/__init__.py", line 1, in <module>
    from .foundations_models import mace_anicc, mace_mp, mace_off
  File "/lustre/home/ah1365/.local/lib/python3.9/site-packages/mace/calculators/foundations_models.py", line 10, in <module>
    from .mace import MACECalculator
  File "/lustre/home/ah1365/.local/lib/python3.9/site-packages/mace/calculators/mace.py", line 24, in <module>
    from mace.cli.convert_e3nn_cueq import run as run_e3nn_to_cueq
  File "/lustre/home/ah1365/.local/lib/python3.9/site-packages/mace/cli/convert_e3nn_cueq.py", line 8, in <module>
    from mace.modules.wrapper_ops import CuEquivarianceConfig
  File "/lustre/home/ah1365/.local/lib/python3.9/site-packages/mace/modules/__init__.py", line 5, in <module>
    from .blocks import (
  File "/lustre/home/ah1365/.local/lib/python3.9/site-packages/mace/modules/blocks.py", line 22, in <module>
    from mace.tools.compile import simplify_if_compile
  File "/lustre/home/ah1365/.local/lib/python3.9/site-packages/mace/tools/compile.py", line 6, in <module>
    import torch._dynamo as dynamo
  File "/lustre/home/ah1365/.local/lib/python3.9/site-packages/torch/_dynamo/__init__.py", line 3, in <module>
    from . import convert_frame, eval_frame, resume_execution
  File "/lustre/home/ah1365/.local/lib/python3.9/site-packages/torch/_dynamo/convert_frame.py", line 33, in <module>
    from torch._dynamo.symbolic_convert import TensorifyState
  File "/lustre/home/ah1365/.local/lib/python3.9/site-packages/torch/_dynamo/symbolic_convert.py", line 30, in <module>
    from . import config, exc, logging as torchdynamo_logging, trace_rules, variables
  File "/lustre/home/ah1365/.local/lib/python3.9/site-packages/torch/_dynamo/trace_rules.py", line 46, in <module>
    from .variables import (
  File "/lustre/home/ah1365/.local/lib/python3.9/site-packages/torch/_dynamo/variables/__init__.py", line 2, in <module>
    from .builtin import BuiltinVariable
  File "/lustre/home/ah1365/.local/lib/python3.9/site-packages/torch/_dynamo/variables/builtin.py", line 53, in <module>
    from .ctx_manager import EventVariable, StreamVariable
  File "/lustre/home/ah1365/.local/lib/python3.9/site-packages/torch/_dynamo/variables/ctx_manager.py", line 22, in <module>
    from .functions import (
  File "/lustre/home/ah1365/.local/lib/python3.9/site-packages/torch/_dynamo/variables/functions.py", line 33, in <module>
    from torch.distributed.fsdp._fully_shard import _fsdp_param_group
  File "/lustre/home/ah1365/.local/lib/python3.9/site-packages/torch/distributed/fsdp/__init__.py", line 1, in <module>
    from ._flat_param import FlatParameter as FlatParameter
  File "/lustre/home/ah1365/.local/lib/python3.9/site-packages/torch/distributed/fsdp/_flat_param.py", line 47, in <module>
    from ._fsdp_extensions import (
  File "/lustre/home/ah1365/.local/lib/python3.9/site-packages/torch/distributed/fsdp/_fsdp_extensions.py", line 6, in <module>
    from torch.distributed._shard.sharded_tensor.api import ShardedTensor
  File "/lustre/home/ah1365/.local/lib/python3.9/site-packages/torch/distributed/_shard/__init__.py", line 1, in <module>
    from .api import _shard_tensor, load_with_process_group, shard_module, shard_parameter
  File "/lustre/home/ah1365/.local/lib/python3.9/site-packages/torch/distributed/_shard/api.py", line 9, in <module>
    from torch.distributed._shard.sharded_tensor import ShardedTensor
  File "/lustre/home/ah1365/.local/lib/python3.9/site-packages/torch/distributed/_shard/sharded_tensor/__init__.py", line 8, in <module>
    from .api import (
  File "/lustre/home/ah1365/.local/lib/python3.9/site-packages/torch/distributed/_shard/sharded_tensor/api.py", line 31, in <module>
    from .reshard import reshard_local_shard, reshuffle_local_shard
  File "/lustre/home/ah1365/.local/lib/python3.9/site-packages/torch/distributed/_shard/sharded_tensor/reshard.py", line 14, in <module>
    from torch.distributed.nn.functional import all_to_all, all_to_all_single
  File "/lustre/home/ah1365/.local/lib/python3.9/site-packages/torch/distributed/nn/__init__.py", line 7, in <module>
    from .api.remote_module import RemoteModule
  File "/lustre/home/ah1365/.local/lib/python3.9/site-packages/torch/distributed/nn/api/remote_module.py", line 42, in <module>
    instantiator.instantiate_non_scriptable_remote_module_template()
  File "/lustre/home/ah1365/.local/lib/python3.9/site-packages/torch/distributed/nn/jit/instantiator.py", line 154, in instantiate_non_scriptable_remote_module_template
    return _do_instantiate_remote_module_template(generated_module_name, str_dict, True)
  File "/lustre/home/ah1365/.local/lib/python3.9/site-packages/torch/distributed/nn/jit/instantiator.py", line 88, in _do_instantiate_remote_module_template
    _write(out_path, generated_code_text)
  File "/lustre/home/ah1365/.local/lib/python3.9/site-packages/torch/distributed/nn/jit/instantiator.py", line 72, in _write
    with open(out_path, "w") as f:
OSError: [Errno 28] No space left on device: '/tmp/tmp5zmkiqrz/_remote_module_non_scriptable.py'
